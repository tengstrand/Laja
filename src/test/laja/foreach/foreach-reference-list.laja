#write "target/test-output/foreach-reference-list.out"
  #set (list = [ "A", "B", "C" ])
  #foreach (x in list)
value = {x}
  #end
#end