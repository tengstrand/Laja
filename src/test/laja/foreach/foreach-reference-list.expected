value = A
value = B
value = C
